lib/payment_calculator/ecf/output_payment_retention_event.rb

```diff
@@ -9,7 +9,7 @@ class OutputPaymentRetentionEvent
 
       def call(total_participants:, event_type:, band:)
         {
-          retained_participants: total_participants,
+          retained_participants: band.number_of_participants_in_this_band(total_participants),
```

Comment:
it doesn't really matter but I'd probably have done this bugfix as a separate PR as it's unrelated to uplift I think